client/src/views/Word.vue

```diff
@@ -25,14 +27,24 @@
 <script>
 import { mapState } from "vuex";
 import TranslateButton from "./../components/TranslateButton";
-
 export default {
   name: "Word",
   components: {
     TranslateButton
   },
   computed: {
-    ...mapState(["dailyData", "loading"])
+    ...mapState(["dailyData", "loading"]),
+    calcualteFontSizeForDailyWord() {
+      if (this.dailyData && this.dailyData.word && this.dailyData.word.word && this.dailyData.word.word.length >= 20) {
```

Comment:
Good use of placing this inside `computed`, maybe a `switch` case would make this more readable?

`switch (this.dailyData.word.word.length)` can work. Then wrap the entire switch statement in an `if (!!this.dailyData.word.word)` which should only evaluate to true if the value is defined.

Reply:
I completely forgot you could use !! to simplify things like that. Thanks for the tip! I also replaced the if elses with a switch case as well. I think it's definitely more readable now.